examples/burial-form/PlotAllowance.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useEffect } from 'react';
+import {
+  TextField,
+  DateField,
+  DebuggerView,
+  FullNameField,
+  Page,
+  RadioGroup
+} from '@department-of-veterans-affairs/va-forms-system-core';
+import { useFormikContext } from 'formik';
+
+export default function PlotAllowance(props) {
+
+  const formikContext = useFormikContext()
+  
+  return (
+    <div className="vads-u-margin-x--1p5">
+      <Page {...props} nextPage="/" prevPage="/military-history/service-periods">
```

Comment:
LGTM 👍 
If you could update Previous url to `/benefits/burial-allowance` and Next url to `/claimant-contact-information`